test/liquid_voting_web/absinthe/mutations/create_delegation/existing_delegations_test.exs

```diff
@@ -75,17 +86,26 @@ defmodule LiquidVotingWeb.Absinthe.Mutations.CreateDelegation.ExistingDelegation
       }
       """
 
-      {:ok, %{data: %{"createDelegation" => original_delegation}}} =
-        Absinthe.run(query, Schema, context: %{organization_id: @organization_id})
+      {:ok, %{data: %{"createDelegation" => updated_delegation}}} =
+        Absinthe.run(query, Schema,
+          context: %{organization_id: proposal_delegation.organization_id}
+        )
+
+      assert updated_delegation["delegator"]["email"] == proposal_delegation.delegator.email
+      assert updated_delegation["delegate"]["email"] == another_delegate.email
+      assert updated_delegation["proposalUrl"] == proposal_delegation.proposal_url
```

Comment:
I'd also assert the delegation id is still the same, as this is something the API consumer might care about when handling the delegation data on the frontend

Reply:
done